### PR TITLE
e2e support mutiple clusters

### PR DESCRIPTION
**Pull request by weekface**
fixes: #159

### CODE REVIEW

File: tests/e2e/e2e_test.go [hunk 1]
```diff
@@ -53,14 +54,12 @@ var _ = SynchronizedBeforeSuite(func() []byte {
...
-		testCreate()
-	})
-	It("should upgrade a tidb cluster", func() {
-		testUpgrade()
-	})
-	It("should scale in/out a tidb cluster", func() {
-		testScale()
-	})
-
+	for i := 0; i < len(fixtures); i++ {
```
Comment (tennix) [inline tests/e2e/e2e_test.go]:
```suggestion
	for _, fixture := range fixtures {
```

Comment (weekface) [reply tests/e2e/e2e_test.go]: This is a golang common mistake: https://github.com/golang/go/wiki/CommonMistakes
Should not use the `for ... range` style.

Comment (tennix) [reply tests/e2e/e2e_test.go]: Oh, I made this mistake too

File: images/tidb-operator-e2e/tidb-operator-values.yaml [hunk 2]
```diff
@@ -50,4 +50,4 @@ scheduler:
     requests:
       cpu: 80m
       memory: 50Mi
-  kubeSchedulerImage: quay.io/coreos/hyperkube:v1.10.4_coreos.0
+  kubeSchedulerImage: 127.0.0.1:5000/pingcap/hyperkube:v1.10.4_coreos.0
```
Comment (tennix) [inline images/tidb-operator-e2e/tidb-operator-values.yaml]: The DinD uses `mirantis/hypokube:final` which contains hyperkube binary.

**Pull request merged**